### PR TITLE
Support create key pair command

### PR DESCRIPTION
**Pull request by zymap**
---

*Motivation*

Support create key pair using RSA and ECDSA signature
algorithm.

*Modifications*

- Add command `create-key-pair`
- Add test for the command
- Add RSA and ECDSA signature algorithm
- Follow the kubectl description style 

*Command output*
```
➜  pulsarctl git:(create-key-pair) ./pulsarctl token create-key-pair -h
USED FOR:
    This command is used for creating a private and public key pair.

REQUIRED PERMISSION:
    This command does not need any permission.

OUTPUT:
    #Successfully create a key pair.
    The private key and public key are generated to (private-key-path) and (public-key-path) successfully.

    #Write a private key to a file was failed.
    [✖]  failed to write private key to the file (private-key-path)

    #Write a public key failed to a file was failed.
    [✖]  failed to write public key to the file (public-key-path)

    #The specified output key file path is empty.
    [✖]  the private key file path and the public key file path can not be empty

Usage: pulsarctl token create-key-pair [flags]

Examples:
    #Create a private and public key pair using RS256 signature algorithm.
    pulsarctl token create-key-pair --output-private-key (filepath) --output-public-key (filepath)

    #Create a private and public key pair using the specified signature algorithm.
    pulsarctl toke create-key-pair --signature-algorithm (algorithm) --output-private-key (filepath) --output-public-key (filepath)



Create key pair flags:
  -a, --signature-algorithm string   The signature algorithm is used for generating the key pair. Valid options are: 'RS256', 'RS384', 'RS512', 'ES256', 'ES384', 'ES512'. (default "RS256")
      --output-private-key string    The file that the private key is written to. (default "private.key")
      --output-public-key string     The file that the public key is written to. (default "public.key")

```

### CODE REVIEW

**Comment by zymap** [comment]
@sijie PTAL. Thanks.

**Comment by zymap** [comment]
@sijie PTAL. Thanks.

File: pkg/ctl/token/create_key_pair.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+	var out []cmdutils.Output
+	defaultOutput := cmdutils.Output{
+		Desc: "Create a key pair successfully.",
+		Out: "The private key and public key are generated to (private-key-path) and " +
+			"(public-key-path) successfully.",
+	}
+
+	writePrivateKeyFailed := cmdutils.Output{
+		Desc: "Write a private key to a file was failed.",
```
Comment (sijie) [inline pkg/ctl/token/create_key_pair.go]:
```suggestion
		Desc: "Failed to write a private key to a file.",
```

File: pkg/ctl/token/create_key_pair.go [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+		Out:  "[✖]  failed to write private key to the file (private-key-path)",
+	}
+
+	writePublicKeyFailed := cmdutils.Output{
+		Desc: "Write a public key failed to a file was failed.",
```
Comment (sijie) [inline pkg/ctl/token/create_key_pair.go]: "Failed to write ..." or "Writing ... was failed"

**Pull request merged**